btc-withdrawal/src/main/kotlin/com/d3/btc/withdrawal/provider/UTXOProvider.kt

```diff
@@ -286,11 +156,10 @@ class UTXOProvider(
 
     // Checks if fee output was addressed to available address
     protected fun isAvailableOutput(
-        availableAddresses: Set<String>,
         output: TransactionOutput
     ): Boolean {
         val btcAddress = outPutToBase58Address(output)
-        return availableAddresses.contains(btcAddress)
+        return btcAddressStorage.isChangeAddress(btcAddress) || btcAddressStorage.isOurClient(btcAddress)
```

Comment:
I think I have seen this statement before 👀

Reply:
ok